dts/bindings/i2s/nordic,nrf-tdm.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic TDM (Time division multiplexed audio interface)
+
+compatible: "nordic,nrf-tdm"
+
+include: [i2s-controller.yaml, pinctrl-device.yaml, nordic-clockpin.yaml, "memory-region.yaml"]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  mck-frequency:
+    type: int
+    description: |
+      Frequency of the MCK clock. Configured independently of SCK.
+
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  easydma-maxcnt-bits:
+    type: int
+    required: true
+    description: |
+      Maximum number of bits available in the EasyDMA MAXCNT register. This
+      property must be set at SoC level DTS files.
+
+  clock-source:
+    type: string
+    default: "FLL16M"
+    description: |
+      Clock source to be used by the TDM peripheral. The following options
+      are available:
+      - "FLL16M": 16 MHz peripheral clock
+      - "ACLK": Audio PLL clock with configurable frequency (frequency for
+        this clock must be set via the "hfclkaudio-frequency" property
+        in the "nordic,nrf-clock" node); this clock source is only available
+        in the nRF53 Series SoCs and it requires the use of HFXO
```

Comment:
What's the point of having ACLK mentioned here when it is only available on nRF5340 which does not have TDM peripheral (nRF5340 has I2S supported by existing driver)?

Reply:
It is available also here, but we don't have the support to request it yet

Reply:
What is "here" and "support to request"?